extra/linux/io.alacritty.Alacritty.appdata.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016-2019 Joe Wilm, The Alacritty Project Contributors -->
+<component type="desktop-application">
+  <id>io.alacritty.Alacritty</id>
+  <!-- Translators: The application name -->
+  <name>Alacritty</name>
+  <launchable type="desktop-id">io.alacritty.Alacritty.desktop</launchable>
+  <project_license>APACHE-2.0</project_license>
+  <metadata_license>APACHE-3.0</metadata_license>
+  <!-- Translators: The application's summary / tagline -->
+  <summary>A cross-platform, GPU enhanced terminal emulator</summary>
+  <description>
+    <p>Alacritty is the fastest terminal emulator in existence. Using the GPU for rendering enables optimizations that simply aren't possible without it.</p>
+    <p>Alacritty currently supports macOS, Linux, BSD, and Windows.</p>
```

Comment:
We should use the manpage's description here:
https://github.com/jwilm/alacritty/blob/master/extra/alacritty.man#L7-L11

This is also the `About` section in the README with some small changes to remove all links.

Reply:
Ok, no problem.